lib/smart_answer_flows/student-finance-calculator/questions/how_many_credits_will_you_study.govspeak.erb

```diff
@@ -0,0 +1,11 @@
+<% content_for :title do %>
+  How many credits will you study in <%= calculator.course_start %>?
```

Comment:
How does this read? I assume:

```
 How many credits will you study in 2018-2019?
```

Reply:
Yes. I'm double-checking this with the content designer.